designs/senya/src/base.mjs

```diff
@@ -0,0 +1,219 @@
+import { waistlineHeight } from './shared.mjs'
+
+function infinitelyHigh(xCoordinate, Point) {
+  return new Point(xCoordinate, -100)
+}
+
+function infinitelyLow(xCoordinate, Point) {
+  return new Point(xCoordinate, 100)
+}
+
+function halfway(measurement, ease) {
+  return measurement / 2 + ease
+}
+
+function notHigherThan(a, b, Point) {
+  if (a.y < b.y) {
+    return new Point(a.x, b.y)
+  }
+  return a
+}
+
+function pointAtX(x, point, Point) {
+  return new Point(x, point.y)
+}
+
+function centerBack(measurements, options, Point) {
+  var hps = new Point(0, 0)
```

Comment:
No need for variable declarations here

```mjs
return { 
  hps: new Point(0, 0),
  neck: new Point(0, options.backNeckCutout * measurements.neck),
  waistline: new Point(
    0,
    waistlineHeight(
      measurements.waistToUnderbust,
      measurements.hpsToWaistBack,
      options.lowerWaistline
    )
  )
}
```

With fat-arrow notation, you can even forego the `return` keyword:

```mjs
const centerBack = (measurements, options, Point) => ({
  hps: new Point(0, 0),
  neck: new Point(0, options.backNeckCutout * measurements.neck),
  waistline: new Point(
    0,
    waistlineHeight(
      measurements.waistToUnderbust,
      measurements.hpsToWaistBack,
      options.lowerWaistline
    )
  )
})
```

Reply:
Went with the former, I prefer my functions being functions not constant lambdas (at least semantically, whether there is any difference under the hood I have no idea :P).